src/Bexio/Provider.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace SocialiteProviders\Bexio;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    const IDENTIFIER = 'BEXIO';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = ['openid profile'];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getAuthUrl($state)
+    {
+        return $this->buildAuthUrlFromBase('https://idp.bexio.com/authorize', $state);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getTokenUrl()
+    {
+        return 'https://idp.bexio.com/token';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getUserByToken($token)
+    {
+        $response = $this->getHttpClient()->get('https://idp.bexio.com/userinfo', [
+            'headers' => [
+                'Authorization' => 'Bearer '.$token,
+            ],
+        ]);
+
+        return json_decode($response->getBody(), true);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function mapUserToObject(array $user): User
+    {
+        $fullName = [];
+        if (!empty($user['given_name'])) {
+            $fullName[] = $user['given_name'];
+        }
+        if (!empty($user['given_name'])) {
```

Comment:
This if statement is checking if the wrong key is empty.

```suggestion
        if (!empty($user['family_name'])) {
```